app/components/post_body_additional_content/post_body_additional_content.js

```diff
@@ -389,6 +427,13 @@ export default class PostBodyAdditionalContent extends ImageViewPort {
             return attachments;
         }
 
+        if (appsEnabled) {
```

Comment:
Apps fall under the same category as MessageAttachments  from my point of view. Since MessageAttachments are handled here, I handled the apps here too. Not sure if MessageAttachments was misplaced from the beginning.